src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,11 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+
+        return numbers.stream().flatMap(n -> Arrays.stream(n.split(",")))
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0).sorted().findFirst()
```

Comment:
```suggestion
                .filter(i -> i % 2 == 0)
                .min(Integer::compareTo)
```
для цього в стрімах якраз є функція мін